layouts/page/event-single.html

```diff
@@ -0,0 +1,27 @@
+{{ define "hero" }}
+    <header class="header-hero">
+        <div class="container mx-auto">
+            <div class="mx-auto text-center">
+                <h1>{{ .Title }}</h1>
+
+            </div>
+        </div>
+    </header>
+{{ end }}
+
+{{ define "main" }}
+    <div class="flex flex-col md:flex-row p-10 md:px-20">
+        <div class="w-full md:w-6/12 lg:w-8/12 mx-auto md:py-12">
+            {{ .Content }}
+        </div>
+        <div class="w-full md:w-6/12 lg:w-4/12 mx-auto">
+            <!-- Calendly inline widget begin -->
+
+            <div class="calendly-inline-widget" data-url="{{ .Params.calendly_url }}" style="min-width:320px;height:750px;"></div>
```

Comment:
Minor nit: Rather than having the inline-style here, can we just add `id="calendly-widget"` and put the styles somewhere in our .CSS file? it just seems like that might be easier to maintain.